low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -549,13 +549,20 @@ class PreUpgradeCmd(Cmd):
 
     def __init__(self, args):
         super().__init__(args)
+        self.level = args.args[0]
 
     def validate(self):
         # Common validator classes to check Cortx/system wide validator
         pass
 
     def process(self):
         logger.info(f"Nothing to be done for {self.name}.")
+        if self.level == 'node':
+            # TODO Perform corresponding actions for node
```

Comment:
Do we know if anything needs to be addressed here? 
If not, then there is no TODO and state no action needed here in code comments

Reply:
Done